common/api-review/auth.api.md

```diff
@@ -81,6 +81,7 @@ export function applyActionCode(auth: Auth, oobCode: string): Promise<void>;
 // @public
 export interface Auth {
     readonly app: FirebaseApp;
+    beforeAuthStateChanged(callback: (user: User | null) => void | Promise<void>): Unsubscribe;

```

Comment:
Did we want to tackle the second argument, the optional `onAbort` callback in this PR? 

That is just a `() => void` that is called useful if `beforeAuthStateChanged` is called multiple times and a subsequent callback rejects the state change. This gives developers a chance to walk back any side effects. 

Useful to me as I integrate with the web frameworks work, for authenticated server-context. onAbort would allow me the chance to destroy the just minted cookie, if the developer is using beforeAuthStateChanged too and rejected the sign in.